app/src/main/java/com/hedvig/app/feature/offer/ui/OfferAdapter.kt

```diff
@@ -396,6 +401,25 @@ class OfferAdapter(
                 e { "Invariant detected: ${data.javaClass.name} passed to ${this.javaClass.name}::bind" }
             }
         }
+
+        class Footer(parent: ViewGroup) : ViewHolder(parent.inflate(R.layout.offer_footer)) {
+            private val binding by viewBinding(OfferFooterBinding::bind)
+            override fun bind(
+                data: OfferModel,
+                fragmentManager: FragmentManager,
+                tracker: OfferTracker,
+                removeDiscount: () -> Unit,
+                marketManager: MarketManager
+            ) {
+                if (data is OfferModel.Footer) {
+                    binding.chatButton.setHapticClickListener {
+                        it.context.startActivity(ChatActivity.newInstance(it.context, true))
+                    }
+                    val link = itemView.context.getString(R.string.OFFER_FOOTER_GDPR_INFO, data.url)
```

Comment:
Does this text key contain `HTML` now? 🤔

Reply:
Yes unfortunately. Is there a better way of handling this?